docs/src/main/asciidoc/security-cors.adoc

```diff
@@ -4,68 +4,91 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 [id="security-cors"]
-= Cross-origin resource sharing
+= Cross-origin resource sharing (CORS)
 include::_attributes.adoc[]
-:diataxis-type: concept
+:diataxis-type: reference
 :categories: security,web
-:keywords: cors,http
+:summary: This guide covers enabling and configuring the CORS filter, which uses HTTP headers to securely manage cross-origin browser requests.
+:keywords: cors,http,configuration, security, headers
 :extensions: io.quarkus:quarkus-vertx-http
 
-Cross-origin resource sharing (CORS) is an HTTP-header-based mechanism that allows a server to indicate any origins other than its own, from which a browser should permit loading resources.
+You can enable and configure the CORS filter, which uses HTTP headers to securely manage cross-origin browser requests.
```

Comment:
@rolfedh I'm not sure it is correct. The filter does not manage the cors requests securily with the help of headers, it is only supports a mechanism to indicate to the browser what can be a safe CORS request... I'm not sure starting the doc with the message what this filter does is better.